airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.json

```diff
@@ -0,0 +1,30 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/amazon-ads",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Amazon Ads Spec",
+    "type": "object",
+    "required": ["client_id", "client_secret", "scope", "refresh_token"],
+    "additionalProperties": false,
+    "properties": {
+      "client_id": {
```

Comment:
This field is not a secret?
Also please add a name to each of 4 properties here. Like `"name": "Client ID",`